### ISSUE TITLE
Train/test speaker separation

### ISSUE THREAD

**Issue by sergeyvilov** [MariaForester/AccentRecognition#1]
Hi,

It seems that you perform train/test split after audio segmentation s.t. some frames from the same speaker may end up in the train set whereas the others are added to the test set. Then, the neural network may simply learn labels for each individual speaker and then be able to carefully reproduce them at the evaluation stage. How do you make sure that your system is generalizable to speakers unseen at training?
